pkg/koordlet/resmanager/cgroup_reconcile_test.go

```diff
@@ -1467,3 +1470,37 @@ func writeMemToCgroupFile(parentDir string, qos *slov1alpha1.ResourceQOS, helper
 	helper.WriteCgroupFileContents(parentDir, system.MemoryPriority, strconv.FormatInt(*qos.MemoryQOS.Priority, 10))
 	helper.WriteCgroupFileContents(parentDir, system.MemoryOomGroup, strconv.FormatInt(*qos.MemoryQOS.OomKillGroup, 10))
 }
+
+//This function is only intended for testing functions within the current file. For specific read/write functionalities, please refer to the executor package.
+func cgroupFileReadIntforTest(cgroupTaskDir string, r system.Resource) (*int64, error) {
+	if supported, msg := r.IsSupported(cgroupTaskDir); !supported {
+		return nil, system.ResourceUnsupportedErr(fmt.Sprintf("read cgroup %s failed, msg: %s", r.ResourceType(), msg))
+	}
+	if exist, msg := system.IsCgroupPathExist(cgroupTaskDir, r); !exist {
+		return nil, system.ResourceCgroupDirErr(fmt.Sprintf("read cgroup %s failed, msg: %s", r.ResourceType(), msg))
+	}
+
+	filePath := r.Path(cgroupTaskDir)
+	klog.V(5).Infof("read %s", filePath)
+
+	data, err := os.ReadFile(filePath)
+	if err != nil {
+		return nil, err
+	}
+	dataStr := strings.Trim(string(data), "\n")
```

Comment:
perhaps `dataStr := strings.Trim(string(data)), "\n ")` is better especially while reading `max  `.